cameras:
  - name: camera_left
    serial_number: "DA2007593"
    exposure:
      mode: 0             # 0: Timed, 1: TriggerWidth
      time_us: 5000       # 曝光时间（us）
      auto: 0             # 0: Off, 1: Once, 2: Continuous
      ae_lower_us: 1000   # 自动曝光下限
      ae_upper_us: 10000  # 自动曝光上限
    gain:
      value_db: 6.0       # 增益值 (dB)
      auto: false         # 是否自动增益
      gain_lower_db: 0.0  # 自动增益下限
      gain_upper_db: 24.0 # 自动增益上限
    white_balance:
      auto: true

  - name: camera_right
    serial_number: "DA5564595"
    exposure:
      mode: 0             # 0: Timed, 1: TriggerWidth
      time_us: 5000       # 曝光时间（us）
      auto: 0             # 0: Off, 1: Once, 2: Continuous
      ae_lower_us: 1000   # 自动曝光下限
      ae_upper_us: 10000  # 自动曝光上限
    gain:
      value_db: 6.0       # 增益值 (dB)
      auto: false         # 是否自动增益
      gain_lower_db: 0.0  # 自动增益下限
      gain_upper_db: 24.0 # 自动增益上限
    white_balance:
      auto: true

loop_rate_hz: 50
